### PR TITLE
Remove illegal use of _collectAllGarbage.

### PR DESCRIPTION
**Pull request by rmacnak-google**
This private RPC is removed.

### CODE REVIEW

**Comment by goderbauer** [comment]
We used that to stabilize some of our benchmarks. Will those become noise again? Or has something else changed that no longer requires this?

File: packages/flutter_driver/lib/src/driver/driver.dart [hunk 1]
```diff
@@ -1031,22 +1030,6 @@ class FlutterDriver {
     return result;
   }
 
-  /// Force a garbage collection run in the VM.
-  Future<void> forceGC() async {
```
[the remark this answers is not in the document]
Comment (rmacnak-google) [reply packages/flutter_driver/lib/src/driver/driver.dart]: Done.

**Comment by rmacnak-google** [comment]
Note that since this code was introduced to flutter_tools, the VM implemented concurrent marking, which should reduce variation in GC latency.

File: packages/flutter_driver/lib/src/driver/driver.dart (hunk 1 above)
[the remark this answers is not in the document]
Comment (goderbauer) [reply packages/flutter_driver/lib/src/driver/driver.dart]: Should we keep the original implementation? Isn't the whole point of marking something as deprecated that you can still get the original behavior? After the next release that contains this change we can remove the method altogether.

**Comment by goderbauer** [comment]
Closing this PR as it hasn't seen any follow-ups in almost a month.

**Comment by goderbauer** [comment]
For future reference: We can totally remove this functionality if needed, but it would have to follow Flutter's breaking change process: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes.

**Pull request closed without merging**